src/api/v4/dataview/histogram/index.js

```diff
@@ -121,18 +161,31 @@ Histogram.prototype._validateBins = function (bins) {
   }
 };
 
+Histogram.prototype._validateStartEnd = function (start, end) {
+  const values = [start, end];
+  const notBothNumber = _.some(values, _.isFinite) && !_.every(values, _.isFinite);
```

Comment:
Shouldn't we check that both numbers are defined and valid, or both are null?

I think we shouldn't allow a single `null` value in this properties as per the comments I read.